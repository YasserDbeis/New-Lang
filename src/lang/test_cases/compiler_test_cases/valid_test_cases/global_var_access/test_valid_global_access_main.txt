str name = "Bob";
int x = 5;
dec y = 10.5;
bool cond = true;

func main()
{
    print(name);
    print(x);
    print(y);
    print(cond);
}