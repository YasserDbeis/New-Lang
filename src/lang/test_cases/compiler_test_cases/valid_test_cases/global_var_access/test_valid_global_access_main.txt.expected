Bob
5
10.5
true